viam-cartographer/scripts/setup_cartographer.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if which brew; then
```

Comment:
why not include this in makefile?

Reply:
And if we want to keep it a `.sh` file, should it still be in the scripts folder

Reply:
It is a script... so yeah, it should be in the scripts folder I think. Not sure I understand why it wouldn't be.

And it's not in the makefile because the rest needs to be a script anyway, but this means it can run standalone (without the makefile) too, since the logic can live in bash anyway. (It's also just cleaner to read all in one place.)

Reply:
I'm fine leaving it in the scripts folder if you think its better there, but we could/should also rename this to just `setup.sh`